lisa/tests/kernel/hotplug/torture.py

```diff
@@ -36,6 +40,55 @@ def __init__(self, plat_info, target_alive, hotpluggable_cpus, live_cpus):
         self.hotpluggable_cpus = hotpluggable_cpus
         self.live_cpus = live_cpus
 
+    @classmethod
+    def _check_cpuhp_seq_consistency(cls, nr_operations, hotpluggable_cpus,
+                                     max_cpus_off, sequence):
```

Comment:
I'm wondering if we actually want this. I don't think we want it in the parent `HotplugTest` class I mentioned above, simply because trying to offline already offline CPUs could actually be a valid test case, for example.